src/components/textarea/TextArea.js

```diff
@@ -423,7 +426,7 @@ export default class TextAreaComponent extends TextFieldComponent {
     // Set the value when the editor is ready.
     this.dataValue = value;
 
-    this.setWysiwygValue(value, flags);
+    this.setWysiwygValue(value, skipSetting, flags);
```

Comment:
Why do we need to pass in a variable? Why not just change the line 

```
if (!skipSetting) {
```

to say

```
if (!_.isEqual(value, this.getValue())) {
```